src/EditorFeatures/CSharp/DecompiledSource/CSharpDecompiledSourceService.cs

```diff
@@ -103,6 +106,11 @@ private Document PerformDecompilation(Document document, string fullName, Compil
 
             // Try to decompile; if an exception is thrown the caller will handle it
             var text = decompiler.DecompileTypeAsString(fullTypeName);
+
+            text += "#if false // " + CSharpEditorResources.Decompilation_log + Environment.NewLine;
```

Comment:
So this will always show at the bottom of the decompiled sources? I wonder if there should be an option, but I'm not sure it matters. Are people copy/pasting the contents and using it elsewhere for any reason? Not going to block on this, and we can always change it later, but I'd like to hear thoughts.